sys-tuner/src/main.rs

```diff
@@ -1,69 +1,6 @@
 use clap::{crate_description, crate_name, value_t_or_exit, App, Arg};
 use log::*;
 
-#[cfg(target_os = "linux")]
-fn tune_poh_service_priority(uid: u32) {
```

Comment:
@carllin let's not delete the function. We might need it later. We can ignore the unused check for now.